tests/gcp/hooks/test_bigquery.py

```diff
@@ -139,98 +140,50 @@ def test_throws_exception_with_incompatible_syntax(self, mock_project_id):
-        self.assertEqual("table", table)
-
-    def test_invalid_syntax_triple_colon(self):
-        with self.assertRaises(Exception) as context:
-            hook._split_tablename('alt1:alt2:alt3:dataset.table',
-                                  'project')
-
-        self.assertIn('Use either : or . to specify project',
-                      str(context.exception), "")
-        self.assertFalse('Format exception for' in str(context.exception))
-
-    def test_invalid_syntax_triple_dot(self):
-        with self.assertRaises(Exception) as context:
-            hook._split_tablename('alt1.alt.dataset.table',
-                                  'project')
-
-        self.assertIn('Expect format of (<project.|<project:)<dataset>.<table>',
-                      str(context.exception), "")
-        self.assertFalse('Format exception for' in str(context.exception))
-
-    def test_invalid_syntax_column_double_project_var(self):
-        with self.assertRaises(Exception) as context:
-            hook._split_tablename('alt1:alt2:alt.dataset.table',
-                                  'project', 'var_x')
-
-        self.assertIn('Use either : or . to specify project',
-                      str(context.exception), "")
-        self.assertIn('Format exception for var_x:',
-                      str(context.exception), "")
-
-    def test_invalid_syntax_triple_colon_project_var(self):
-        with self.assertRaises(Exception) as context:
-            hook._split_tablename('alt1:alt2:alt:dataset.table',
-                                  'project', 'var_x')
-
-        self.assertIn('Use either : or . to specify project',
-                      str(context.exception), "")
-        self.assertIn('Format exception for var_x:',
-                      str(context.exception), "")
-
-    def test_invalid_syntax_triple_dot_var(self):
-        with self.assertRaises(Exception) as context:
-            hook._split_tablename('alt1.alt.dataset.table',
-                                  'project', 'var_x')
-
-        self.assertIn('Expect format of (<project.|<project:)<dataset>.<table>',
-                      str(context.exception), "")
-        self.assertIn('Format exception for var_x:',
-                      str(context.exception), "")
+        with self.assertRaisesRegexp(Exception, "INTERNAL: No default project is specified"):
+            hook._split_tablename("dataset.table", None)
+
+    @parameterized.expand([
+        ("project", "dataset", "table", "dataset.table", "project"),
+        ("alternative", "dataset", "table", "alternative:dataset.table", "project"),
+        ("alternative", "dataset", "table", "alternative.dataset.table", "project"),
+        ("alt1:alt", "dataset", "table", "alt1:alt.dataset.table", "project"),
+        ("alt1:alt", "dataset", "table", "alt1:alt:dataset.table", "project"),
+    ])
+    def test_split_tablename(
+        self, project_expected, dataset_expected, table_expected, table_input, default_project_id
```

Comment:
Is defaullt_project_id a constant or variable value?

Reply:
It could be a constant across all these tests. I will change it.